vcd/resource_vcd_network_isolated_v2_nsxt_test.go

```diff
@@ -122,33 +125,35 @@ resource "vcd_network_isolated_v2" "net1" {
 
   gateway       = "1.1.1.1"
   prefix_length = 24
-
+  guest_vlan_allowed = false
+  
   static_ip_pool {
-	start_address = "1.1.1.10"
-	end_address   = "1.1.1.20"
-  }
-
-  metadata = {
-    {{.MetadataKey}} = "{{.MetadataValue}}"
-  }
+	  start_address = "1.1.1.10"
+	  end_address   = "1.1.1.20"
+	}
+	
+	metadata = {
+		{{.MetadataKey}} = "{{.MetadataValue}}"
+	}
 }
 `
 
 const testAccVcdNetworkIsolatedV2NsxtStep2 = `
 resource "vcd_network_isolated_v2" "net1" {
-  org  = "{{.Org}}"
-  vdc  = "{{.NsxtVdc}}"
-  
-  name        = "{{.NetworkName}}"
-  description = "updated NSX-T isolated network test"
+	org  = "{{.Org}}"
+	vdc  = "{{.NsxtVdc}}"
+	
+	name        = "{{.NetworkName}}"
+	description = "updated NSX-T isolated network test"
 
   gateway       = "1.1.1.1"
   prefix_length = 24
-
+  guest_vlan_allowed = true
+  
   static_ip_pool {
-	start_address = "1.1.1.10"
-	end_address   = "1.1.1.20"
-  }
+	  start_address = "1.1.1.10"
+	  end_address   = "1.1.1.20"
+	}
```

Comment:
For consistency we keep these configuration templates in line with Terraform style guidelines
```suggestion
  start_address = "1.1.1.10"
  end_address   = "1.1.1.20"
}
```